File: ruby/list-seeds.rb
# ItemSeedsst < ItemActual < Item < MemHack::Compound

df.world.items.all.each{ |item|
    if item.getType().to_s == "SEEDS" then
        puts item.getMaterial()
        puts item.getMaterialIndex()
        puts item.getPlantID()
    end
    # puts item.class.to_s
}

File: lua/list-seeds.lua
for _, item in pairs(df.global.world.items.all) do
    if item.pos.x ~= -30000 and
        item:getType() == df.item_type.SEEDS then
            
        print(("%d %d %d"):format(
            item.pos.x, item.pos.y, item.pos.z
        ))
    end
end